lib/msal-browser/src/telemetry/BrowserPerformanceClient.ts

```diff
@@ -223,4 +259,19 @@ export class BrowserPerformanceClient
             manuallyCompleted
         );
     }
+
+    /**
+     * Returns event duration in milliseconds using window performance API if available. Returns undefined otherwise.
+     * @param startTime {DOMHighResTimeStamp | undefined}
+     * @returns {number | undefined}
+     */
+    private getPerfDurationMs(
```

Comment:
This could also probably be a standalone function rather than a class method

Reply:
Made it a standalone function.